src/ProjectNeon/Assets/State/Templates/AdventureProgress.cs

```diff
@@ -17,6 +18,10 @@ class AdventureProgress : ScriptableObject
     public void Reset()
     {
         currentStageIndex = -1;
+        if (currentAdventure.Stages.Length < 1)
+        {
+            Console.WriteLine("The adventure must have a least one stage!");

```

Comment:
In Unity, we don't write to the Console. Use this instead: https://docs.unity3d.com/ScriptReference/Debug.Log.html